### DISCUSSION TITLE
minimal Docker image

### DISCUSSION THREAD

**Discussion by Saphyel** [transmission/transmission#6641] [Ideas]
I was thinking if it would be possible to have like a stable docker image for this project? 

I saw other projects like this one https://github.com/linuxserver/docker-qbittorrent
And I think it could be useful for development point of view because devs could use docker to test the client (without GTK)

And it's useful for users too, I have a raspberry pi as server and I'm moving everything into docker so I don't have to mess with the OS

**Comment by Nemric** [comment]
Hi,
I'm used to update https://hub.docker.com/r/nemric/transmission weekly with tags :
latest : the latest stable actually 4.0.5
main : the latest build with latest commits (unstable)

I do compile from sources on every build, based on latest Alpine
